### ISSUE TITLE
[Spigot] Resource Not Available? Click Here

### ISSUE THREAD

**Issue by Wolfieheart** [Wolfieheart/ArmorStandEditor#288]
**TLDR:**
At the moment, the Spigot URLs for ArmorStandEditor [1.17 -> 1.20/1.20.1] will not work, as I am currently banned from Spigot due to the accidental distribution of the Premium Plugin - Residence.

**More Long Term Timeline**
- ArmorStandEditor 1.20 was released on June 28th 2023 to Spigot / Github / Hangar and Modrinth
- 14 hours after the fact, I got a Private Message on Spigot stating that I was illegally distributing Residence as part of ArmorStandEditor and I had to remove it.
   - I was at work when this happened and my work's VPN blocks all connections to SpigotMC, so I had to tether in an attempt to access the site
- After tethering I noticed that my account was banned within 20 minutes after the DM was received.
- I take the executive decision to remove Residence from ArmorStandEditor completely, therefore dropping support for the project until I found a way to implement it in a way that doesnt require redistribution 
   - More info on: https://github.com/Wolfieheart/ArmorStandEditor/pull/283
   - If you do know how I can do this without dropping it into my resources and redistributing it, then please make a Pull Request and readd in the changes from the above.
- I also remove all the affected releases (1.19.3-39 and Higher) from all distribution platforms
- I email the Spigot Staff asking for an Unban explaining that I have been at work, I have only 20 mins to get it fixed and all the remedial steps outlined above. 
- I reach out to the person who told me about the issue, and tell them it has been fixed and explained that I was banned
- Our read me gets updated to include the fact that Spigot Downloading currently doesnt work.

# Please do not submit any issues about the UpdateChecker saying you need to update and the Spigot Resource not being available - This will be fixed in the next release.
# Please do not submit bugs/new feature requests asking if and when spigot will be back - Current ETA: There is none!

Thanks in advance and to repost something from yesterday:

![image](https://github.com/Wolfieheart/ArmorStandEditor/assets/18635695/70f5d316-dd2a-4305-bc92-62ca6cf4d123)

Kr,
Wolfieheart
